fv_be/app/models/part_of_speech.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+from django.utils.translation import gettext as _
+
+# FirstVoices
+from .base import BaseModel
+
+
+class PartOfSpeech(BaseModel):
+    """Model for Parts Of Speech."""
+
+    # Fields
+    title = models.CharField(max_length=200)
+    parent = models.ForeignKey("self", null=True, blank=True, on_delete=models.PROTECT, related_name="children")
```

Comment:
Parts of speech should have the same "no grandparents" restriction as categories do.